app/src/test/java/com/team7/cmput301/android/theirisproject/CareProviderTest.java

```diff
@@ -1,27 +1,63 @@
 package com.team7.cmput301.android.theirisproject;
 
 import com.team7.cmput301.android.theirisproject.model.CareProvider;
+import com.team7.cmput301.android.theirisproject.model.Patient;
 
 import org.junit.Assert;
 import org.junit.Test;
 
 import static com.team7.cmput301.android.theirisproject.model.User.*;
 
 public class CareProviderTest {
+
+    private String name;
+    private String email;
+    private String phoneNumber;
+    private UserType type;
+
     @Test
     public void testCareProvider() {
         // Test creation of a Care Provider
-        String name = "CareProviderOne";
-        String email = "CareProviderOne@hotmail.com";
-        String phoneNumber = "123-456-789";
-        UserType type = UserType.CARE_PROVIDER;
 
-        CareProvider careProvider = new CareProvider(name, email, phoneNumber);
+        CareProvider careProvider = getTestCareProvider();
 
         Assert.assertEquals(name, careProvider.getName());
         Assert.assertEquals(email, careProvider.getEmail());
         Assert.assertEquals(phoneNumber, careProvider.getPhone());
         Assert.assertEquals(type, careProvider.getType());
     }
 
+    @Test
+    public void testAddPatient() {
+
+        CareProvider careProvider = getTestCareProvider();
+        Patient patient = getTestPatient();
+
+        careProvider.addPatient(patient);
+        Assert.assertEquals(careProvider.getPatients().size(), 1);
+
```

Comment:
Assert the patient you added is the same as the patient you get back out from the list as well

Reply:
Note that this should probably be done for all of the tests where you try adding something